CondFormats/HcalObjects/interface/HcalCombinedRecordsGPU.h

```diff
@@ -0,0 +1,20 @@
+#ifndef CondFormats_HcalObjects_interface_HcalCombinedRecordsGPU_h
+#define CondFormats_HcalObjects_interface_HcalCombinedRecordsGPU_h
+
+#include "CondFormats/DataRecord/interface/HcalPedestalWidthsRcd.h"
+#include "CondFormats/DataRecord/interface/HcalPedestalsRcd.h"
+#include "CondFormats/DataRecord/interface/HcalQIEDataRcd.h"
+#include "CondFormats/DataRecord/interface/HcalQIETypesRcd.h"
```

Comment:
This file is the only one pulling in the dependence on `CondFormats/DataRecord`. I wonder if it would be better to place these Record definitions in other package to avoid that (e.g. in `CondFormats/DataRecord`).